linkerd/docs/client_tls.md

```diff
@@ -34,17 +72,18 @@ tls:
   caCertPath: /foo/caCert.pem
 ```
 
-`io.l5d.static`
+kind: `io.l5d.static`
 
-Use a single common name for all TLS requests.  This assumes that all servers
-that the router connects to all use the same TLS cert (or all use certs
-generated with the same common name).  This plugin supports the following
-options:
+Uses a single common name for all TLS requests.  This assumes all servers
+that the router connects to use the same TLS cert (or all use certs
+generated with the same common name).
 
-* *commonName* -- Required.  The common name to use for all TLS requests.
-* *caCertPath* -- Optional.  Use the given CA cert for common name validation.
+Key | Default Value | Value Description
+--- | ------------- | -----------------
+commonName | _required_ | The common name to use for all TLS requests.
+caCertPath | ? | The path to the CA cert used for common name validation.
```

Comment:
There is no default value, supplying a CA cert is optional.  If no CA cert is supplied, then some well-known authority is used, I think.